tests/test_stepper_motor.py

```diff
@@ -0,0 +1,49 @@
+"""Tests for DummyStepperMotor."""
+import pytest
+
+from finesse.hardware.dummy_stepper_motor import DummyStepperMotor
+
+PRESETS = ("zenith", "nadir", "hot_bb", "cold_bb", "home")
+
+
+def test_constructor() -> None:
+    """Check that the constructor only accepts values in permissible range."""
+    with pytest.raises(ValueError):
+        DummyStepperMotor(0)
+
+    with pytest.raises(ValueError):
+        DummyStepperMotor(-1)
+
+    # Should work
+    DummyStepperMotor(len(PRESETS))
+
+
+def test_move_to_number() -> None:
```

Comment:
I think that these tests that check that some values work well but others are not valid call for `pytest.parametrize` or even `hypothesis` to be really thorough a check edge cases.

Reply:
Ah I've never used either of those, but it seems a sensible approach. I've ported over the tests to use `pytest.parametrize` now.